pkg/asset/daemonset.go

```diff
@@ -70,7 +70,7 @@ func (d *daemonset) New() *appsv1.DaemonSet {
 								"--bind-address=127.0.0.1",
 								"--tls-cert-file=/var/serving-cert/tls.crt",
 								"--tls-private-key-file=/var/serving-cert/tls.key",
-								"--v=3",
+								"--v=5",
```

Comment:
If this change is not related to the Selinux workaround, consider moving to a separate PR to bump the default verbosity of the operand.

Reply:
fixed!